src/pages/product-page/index.tsx

```diff
@@ -10,10 +10,16 @@ import IconArrowRight from '../../icons/arrowright.svg?react'
 import { ModalProductLimits } from '../../components/modal-product-limits/ModalProductLimits'
 import { ProductResponse } from '../../types'
 import axiosInstance from '../../axios/settings'
-import { useSelector } from 'react-redux'
-import { RootState } from '../../redux/store'
-import { Notify } from 'notiflix'
+import { useDispatch, useSelector } from 'react-redux'
+import { AppDispatch, RootState } from '../../redux/store'
+import Notiflix, { Notify } from 'notiflix'
 import { formatWeight } from '../../helpers/formatWeightToString'
+import { fetchBasketItemsThunk } from '../../redux/basket-item-count/operations'
+import { updateCount } from '../../redux/basket-item-count/slice'
+
+Notiflix.Notify.init({
+  position: 'center-top'
+})
```

Comment:
why do you init it in this component?

Reply:
cause these are temporary notifications and default settings interfere with testing